### PR TITLE
doc: add tiup Quick Start in README.md

### PR DESCRIPTION
**Pull request by zwj-coder**
### What problem does this PR solve? 
Add description about how to use tiup to test the backup && restore in Tidb cluster.

### What is changed and how it works?
1. Use tiup to build Tidb cluster
2. Use tiup bench to generate test data
3. Backup these cluster via br
4. Remove corresponding data
5. Use br restore cluster

### Check List 

Tests
 - Manual test (add detailed scripts or steps blew)

### Release Note

 - No release note.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,42 @@ bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
...
+	-s local:///mnt/d/backup_test/ \
+	--pd ${PD_ADDR}:2379 \
+	--log-file backup_test.log 
+
+# Let's drop the table.
+mysql -uroot --host 127.0.0.1 -P4000 -E -e "USE test; DROP TABLE order_line; show tables" -u root -p
+
+# Restore from the backup.
+bin/br restore table --db test \
+    --table oder_line \
```
Comment (YuJuncen) [inline README.md:117]: Seems there is a typo.
```suggestion
    --table order_line \ 
```

File: README.md [hunk 2]
```diff
@@ -87,6 +87,42 @@ bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
...
+
+```sh
+# Using tiup to start a TiDB cluster.
+tiup playgroup
+
+# Using tiup bench to generater test data.
+tiup bench tpcc --warehouses 1 prepare
+
+# How many row do we get? 300315 rows.
+mysql --host 127.0.0.1 --port 4000 -E -e "SELECT COUNT(*) FROM test.customer" -u root -p
```
Comment (YuJuncen) [inline README.md:100]: If we selected from `test.customer` before backing up, then select from `test.order_line` after restoration, we would probably get unexpected result.

File: README.md [hunk 3]
```diff
@@ -87,6 +87,42 @@ bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
...
+
+# Build br.
+make build
+
+# Backup TPC-C test data.
+bin/br backup table --db test \
+    --table order_line \
+	-s local:///mnt/d/backup_test/ \
```
Comment (YuJuncen) [inline README.md:108]: How about backup to `/tmp/backup_test`? This path seems only exists at WSL environment.

**Review by YuJuncen — COMMENTED** [review]
Rest looks good to me.

File: README.md (hunk 2 above)
Comment (zwj-coder) [reply README.md:100]: > If we selected from `test.customer` before backing up, then select from `test.order_line` after restoration, we would probably get unexpected result.

I'm sorry, this is a typo, I should check more carefully. `test.customer` should be corrected to `test.order_line`.

Comment (YuJuncen) [reply README.md:100]: Never mind, typo is a component of life...

Comment (zwj-coder) [reply README.md:100]: > Never mind, typo is a component of life...

I will redo a backup && restore test quickly and correct my pr.

File: README.md [hunk 4]
```diff
@@ -87,6 +87,42 @@ bin/br restore full --pd pd0:2379 --storage "s3://mybucket/full" \
     --s3.endpoint="$S3_ENDPOINT"
 ```
 
+## Tiup Quick Start
```
Comment (3pointer) [inline README.md:90]:
```suggestion
## Quick Start(tiup)
```

Comment (3pointer) [inline README.md:90]: also you can change L41 to `## Quick start(docker-compose)`

**Review by 3pointer — COMMENTED** [review]
Rest lgtm

**Comment by kennytm** [comment]
@zwj-coder hi, the bot requires the [CLA to be signed](https://github.com/pingcap/br/pull/1038#issuecomment-823003828) before merging ("zhangwenjun1997" refers to the author email used in the commit, and "zwj-coder" refers to the GitHub account). Could you resolve this? Thanks!

**Pull request merged**